src/analysis_and_optimization/Mir_utils.ml

```diff
@@ -61,7 +61,7 @@ let trans_bounds_values (trans : Expr.Typed.t Transformation.t) : bound_values =
        until we define a distribution which accepts a tuple,
        this doesn't matter.
     *)
-   |TupleTransformation _ ->
+   |TupleTransformation _ | StochasticColumn | StochasticRow ->
```

Comment:
These can actually be up with `Simplex` since they are bound between 0 and 1, I think?